<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RBasicDialog</class>
 <widget class="QDialog" name="RBasicDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font:14px &quot;宋体&quot;;
}
#titleArea{
border:1px solid rgb(15,67,108);
border-bottom:0px;
border-top-left-radius:5px;
border-top-right-radius:5px;
background-color:rgb(1,101,178);
min-height:28px;
max-height:28px;
}
#contentArea{
border:1px solid rgb(15,67,108);
border-top:0px;
border-bottom-left-radius:5px;
border-bottom-right-radius:5px;
background-color:rgb(1,101,178);
}
#titleLabel{
color:white;
}
#content{
background-color:rgb(207,235,255);
}
#contentDialog{
background-color:transparent;
}
#buttonArea{
max-height:50px;
background-color:transparent;
}
#buttonArea QPushButton{
min-width:80px;
max-width:80px;
min-height:25px;
max-height:25px;
}
#btnClose{
min-width:40px;
min-height:20px;
border:1px solid rgb(15,67,108);
border-radius:1px;
color:rgb(163,217,255);
padding:-1px 0 1px 0;
font: 20px &quot;Microsoft YaHei UI&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(19, 152, 253), stop:0.15 rgb(1, 101, 178), stop:0.85 rgb(1, 101, 178), stop:1 rgb(5, 112, 195));
}
#btnClose:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(19, 152, 253), stop:0.15 rgb(59, 135, 194), stop:0.85 rgb(59, 139, 194), stop:1 rgb(5, 112, 195));
}
#btnClose:pressed{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(19, 152, 253), stop:0.15 rgb(1, 83, 147), stop:0.85 rgb(1, 83, 147), stop:1 rgb(5, 112, 195));
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleArea" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>29</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>29</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnClose">
          <property name="text">
           <string>×</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="contentArea" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QWidget" name="content" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QDialog" name="contentDialog"/>
         </item>
         <item>
          <widget class="QWidget" name="buttonArea" native="true">
           <layout class="QHBoxLayout" name="buttonLayout"/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
